pkg/gather/clusterconfig/operators.go

```diff
@@ -92,28 +96,120 @@ func GatherClusterOperators(g *Gatherer, c chan<- gatherResult) {
 		c <- gatherResult{nil, []error{err}}
 		return
 	}
-	records, errors := gatherClusterOperators(g.ctx, gatherConfigClient, gatherKubeClient.CoreV1(), discoveryClient, dynamicClient)
-	c <- gatherResult{records, errors}
+	records, errs := gatherClusterOperators(g.ctx, gatherConfigClient, gatherKubeClient.CoreV1(), discoveryClient, dynamicClient)
+	c <- gatherResult{records, []error{errs}}
```

Comment:
This is wrong. We shouldn't send the `error` array when the `errs` is `nil`.

Reply:
Nice catch. Thanks.